libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -319,6 +322,82 @@ void AP_UAVCAN::init(uint8_t driver_index, bool enable_filters)
     debug_uavcan(2, "UAVCAN: init done\n\r");
 }
 
+// send ESC telemetry messages over MAVLink
+void AP_UAVCAN::send_esc_telemetry_mavlink(uint8_t mav_chan)
+{
+    static const uint8_t MAV_ESC_GROUPS = 3;
+    static const uint8_t MAV_ESC_PER_GROUP = 4;
```

Comment:
This is AKA `MAVLINK_MSG_ESC_TELEMETRY_1_TO_4_FIELD_VOLTAGE_LEN` - and several other equivalent defines.

I don't think using those would really add much here.

Reply:
Not sure I understand. Are suggesting I should use the existing MAVLINK_MSG_XXX defines, or use non of those and replace them with number literals instead?

Reply:
Then let's consider this done